agent/agentssh/agentssh_test.go

```diff
@@ -96,7 +96,7 @@ func TestNewServer_ExecuteShebang(t *testing.T) {
 	t.Run("Args", func(t *testing.T) {
 		t.Parallel()
 		cmd, err := s.CreateCommand(ctx, `#!/usr/bin/env bash
-		echo test`, nil)
+		echo test`, nil, nil)
```

Comment:
Perhaps we should add a test for a custom implementation as well?

Reply:
Can do!